.github/workflows/standard.yml

```diff
@@ -17,8 +17,8 @@ jobs:
       # Extras that will be installed for both conda/pip:
       testing-extras: "pcdsdevices pcdsutils"
       # Extras to be installed only for conda-based testing:
-      conda-testing-extras: "line_profiler<4.0.0"
+      conda-testing-extras: ""
       # Extras to be installed only for pip-based testing:
-      pip-testing-extras: "line-profiler<4.0.0 wheel"
+      pip-testing-extras: "wheel"
```

Comment:
Doesn't it? 
[new log](https://github.com/pcdshub/happi/actions/runs/4068015539/jobs/7006060077#step:12:135): creates wheel for sentinels
[old log](https://github.com/pcdshub/happi/actions/runs/4059930110/jobs/6988509834#step:12:147):

Reply:
Check the step before it in 'new log', though: https://github.com/pcdshub/happi/actions/runs/4068015539/jobs/7006060077#step:11:340
(Perhaps that's a separate issue and means we need 'wheel' also in our pyproject.toml build section?)

Reply:
If I understand this all correctly, wheel needs to be present before any pip installations occur in order to avoid this error.  Currently we install all the testing dependencies in one go, so we can: 
- install wheel in a prior step
  - in the ["Upgrade pip"](https://github.com/pcdshub/pcds-ci-helpers/blob/97cea13ebe9d3cd2db451571cd0eacd2f218b8f2/.github/workflows/python-pip-test.yml#L87) or ["build wheel and source distribution"](https://github.com/pcdshub/pcds-ci-helpers/blob/97cea13ebe9d3cd2db451571cd0eacd2f218b8f2/.github/workflows/python-pip-test.yml#L91) steps?  (these are the ones that made sense to me at first glance)
- install the test dependencies with the `--pep-517` pip option

All of this is driven by the impending [PEP517](https://peps.python.org/pep-0517/) and an effort to make all packages installable via wheels ([pip deprecation warning issue](https://github.com/pypa/pip/issues/8559))

I do think this can be pushed out of the scope of this PR.  I'll make an issue with the above text for the ci-helpers, since I'm not fully sure which approach to take